modules/media/help/media.md

```diff
@@ -0,0 +1,17 @@
+# Media
+
+The Media module serves as a repository for files associated with a particular candidate-timepoint or instrument in a study.  Users can access and upload files such as PDF scans, recordings, log files, or stimulus presentation media in formats such as .pdf, .mp4, .mp3, and .txt. 
+
+## Searching for a Media File
+Under the Browse tab, use the Selection Filters to search for files by fields such as including filename, file type, candidate PSCID, Visit Label, Instrument, Site, or name of user who uploaded the file.  Partial string matching can be used on many fields (e.g. filename).  As filters are selected, the below data table will dynamically update with relevant results.  Click the “Clear Filters” button to reset all filters. 

```

Comment:
"including" shouldn't be there in the first sentence. The capitalization of what you can filter by is also inconsistent (I think they should all be lower case, they're not proper nouns.)

Reply:
thanks, good catch.  In general, where a Selection Filter field is referenced by its label (usually capitalized), that will be reflected in the help text for easy identification by the user.